canonical-websites/snapcraft.io
# Remove raven cdn

# Summary

- Remove raven cdn and use [npm package](https://www.npmjs.com/package/raven-js)
Fixes #497 

# QA

- `./run`
- install umatrix
- go on http://0.0.0.0:8004/toto
- Make sure the map is well diplayed
